source/administration/config-settings.rst

```diff
@@ -613,25 +613,25 @@ ________
 
 AD/LDAP (Enterprise)
 ```````````````````````````
-Enable sign-in with LDAP
+Enable sign-in with AD/LDAP
```

Comment:
Are we changing the actual System Console settings to say AD/LDAP as well? 

I know we changed the title of the page, but wasn't sure if the settings on the page also got renamed